config.go

```diff
@@ -161,6 +160,11 @@ type RuntimeConfig interface {
 	// When the invocations of api.Function are closed due to this, sys.ExitError is raised to the callers and
 	// the api.Module from which the functions are derived is made closed.
 	WithCloseOnContextDone(bool) RuntimeConfig
+
+	// WtihPerfmap enables perfmap files generation at compile time. Defaults to false.
+	//
+	// Perfmap files are used as source for debug symbols by external profiler such as `perf`.
+	WithPerfmap(bool) RuntimeConfig
```

Comment:
so how about having this as the experimental rather than 1st class API right away which we must promise API stability on? You can define and insert the param in context just like listeners: https://github.com/tetratelabs/wazero/blob/main/experimental/listener.go#L12